docs/sql-ref-syntax-qry-window.md

```diff
@@ -19,4 +19,208 @@ license: |
   limitations under the License.
 ---
 
-**This page is under construction**
+### Description
+
+Similarly to aggregate functions, window functions operate on a group of rows. However, unlike aggregate functions, window functions perform aggregation without reducing, calculating an aggregated value for each row in the specified window. Window functions are useful for processing tasks such as calculating a moving average, computing a cumulative, or accessing the value of rows given the relative position of the current row. Spark SQL supports three types of window functions:
```

Comment:
"without reducing"? Sounds confusing. How about "without reducing the number of rows"?

And "but calculating an aggregated value for each row in the specified window."